build-tools/automation/build.groovy

```diff
@@ -291,7 +291,7 @@ timestamps {
             }
         }
 
-        utils.stageWithTimeout('run all tests', 240, 'MINUTES', XADir, false) {   // Typically takes 1hr and 50 minutes (or 110 minutes)
+        utils.stageWithTimeout('run all tests', 360, 'MINUTES', XADir, false) {   // Typically takes 1hr and 50 minutes (or 110 minutes)
```

Comment:
Should probably update the comment, as if we're bumping to 360 minutes (6h), then it likely *no longer* "typically takes 1hr 50 min"...

Reply:
very true, Done!